examples/gazelle/README.md

```diff
@@ -0,0 +1,5 @@
+This example contains java files which don't have BUILD targets for them.
+
+It requires running `bazel run :gazelle` to generate targets which can be built.
+
+There is a test using `genquery` which asserts that the right targets are generated.
```

Comment:
Does the plugin also automatically add the JUnit dep? Could be worth checking `--output=build` ignoring comment lines.

Reply:
For now I'm happy that we're running the test and it passes, but we could expand this more in the future :)